src/data/actions/dashboardAnalytics.js

```diff
@@ -5,6 +5,7 @@ import {
   CLEAR_DASHBOARD_ANALYTICS,
 } from '../constants/dashboardAnalytics';
 import EnterpriseDataApiService from '../services/EnterpriseDataApiService';
+import * as logging from '../../Logging';
```

Comment:
I would recommend only importing the functions we need in this file, instead of using the wildcard asterisk.